test/api/v3/integration/tasks/PUT-tasks_id.test.js

```diff
@@ -176,6 +177,42 @@ describe('PUT /tasks/:id', () => {
       expect(savedDaily.reminders[0].id).to.equal(id1);
       expect(savedDaily.reminders[1].id).to.equal(id2);
     });
+
+    it('can set a shortName if the shortName does not already exist', async () => {
+      let savedDaily = await user.put(`/tasks/${daily._id}`, {
+        shortName: 'short-name',
+      });
+
+      expect(savedDaily.shortName).to.eql('short-name');
+    });
+
+    it('ignores shortName if it is already set', async () => {
```

Comment:
because I see it as a unique identifier similar to the UUID: it should always point to the same resource. If I have an integration that points to the task with short name `my-integration` I'd like to know that it would always point to that task and that if I use it one year later it'll always be an habit and not have been changed to a daily for example